actions-job.php

```diff
@@ -1,4 +1,34 @@
- <?php include_once('header.php'); // Include Header once ?>
+ <?php include_once('header.php'); // Include Header once
+ 
+ if (isset($_POST['new_job'])) {
+     if ($_SESSION['is_company'] = true) { // Check if the log is company
```

Comment:
You should first check if the key exists in the `$_SESSION` variable. Then do whatever you want with it.
Currently if the key doesn't exist it will throw an error.